resume.tex

```diff
@@ -0,0 +1,123 @@
+
+\input{command.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Now for the actual document:
+
+\begin{document}
+
+\input{header.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%% 
+\begin{ressection}{Employment}
+	\begin{restitledposition}{Nurx / Thirty Madison}{Staff Software Engineer}{April 2022--August 2024}
+		\ressubitem{\textbf{Job Runner} Designed, implemented, and maintained an in-house asynchronous job running
+		solution. Using RabbitMQ, NestJS, and NodeJS/TypeScript, created a new standardized solution for running async
+		tasks at scale across a suite of microservices.}
+		\ressubitem{\textbf{Kafka Topic Isolation} Isolated Kafka topics within staging environments
+		leading to improved stability across dozens of staging environments leading to increased developer efficiency.}
```

Comment:
some HMs like quantified results (dev efficiency by x%). I always ignored them because they're made up anyway.

Reply:
26% is about right